int8 data_flag
int32 data_rate
int16[6] adsGain
float32 vref
float32[6] adsRate
int8[6] inBuf
float32[6] OFC
float32[6] FSC
